src/indieweb_utils/indieauth/flask.py

```diff
@@ -119,7 +119,12 @@ def is_authenticated(token_endpoint: str, headers: dict, session: dict, approved
     else:
         return False
 
-    check_token = requests.get(token_endpoint, headers={"Authorization": "Bearer " + access_token})
+    try:
+        check_token = requests.get(token_endpoint, headers={"Authorization": "Bearer " + access_token}, timeout=5)
```

Comment:
🐼 `f-strings` are a bit more readable.

```suggestion
        check_token = requests.get(token_endpoint, headers={"Authorization": f"Bearer {access_token}"}, timeout=5)
```